package.json

```diff
@@ -4,23 +4,34 @@
         "@jgoz/esbuild-plugin-typecheck": "^3.1.0",
         "@league-of-foundry-developers/foundry-vtt-types": "^9.269.0",
         "@types/node": "^20.8.9",
+        "@typescript-eslint/eslint-plugin": "^6.13.2",
         "@yaireo/tagify": "^4.17.9",
         "buffer": "^6.0.3",
         "chalk": "^5.3.0",
         "del": "^7.1.0",
         "esbuild": "^0.19.5",
+        "eslint": "^8.55.0",
+        "eslint-config-prettier": "^9.1.0",
+        "eslint-config-standard-with-typescript": "^40.0.0",
+        "eslint-plugin-prettier": "^5.0.1",
         "events": "^3.3.0",
         "fs-extra": "^11.1.1",
         "gulp": "^4.0.2",
         "gulp-sass": "^5.0.0",
         "gulp-sourcemaps": "^3.0.0",
         "node-sass": "^9.0.0",
+        "prettier-eslint": "^16.1.2",
         "stream": "^0.0.2",
         "timers": "^0.1.1",
-        "typescript": "^5.2.2",
+        "typescript": "^5.3.2",
```

Comment:
I'll include a reminder to devs in the release notes for this to update their packages `npm install`.